packages/browser/src/plugins/remote-loader/index.ts

```diff
@@ -289,12 +294,7 @@ export async function remoteLoader(
               plugin
             )
 
-            /** Make sure we only apply destination filters to actions of the "destination" type to avoid causing issues for hybrid destinations */
-            if (
-              routing.length &&
-              routingMiddleware &&
-              plugin.type === 'destination'
```

Comment:
Ah cool - so we were actually doing this check twice because `addMiddleware` also checks this